app/views/insured/consumer_roles/_tribe_fields.html.erb

```diff
@@ -85,4 +86,30 @@
     <% end %>
   </div>
 
-</div>
\ No newline at end of file
+</div>
+
+<script>
+    var enroll_state_abbr = $('#enroll_state_abbr').val();
+    if (enroll_state_abbr === 'ME') {
```

Comment:
Same here. Make use of RR configuration instead of comparing these against the hardcoded text.